### ISSUE TITLE
Question for SRN training dataset

### ISSUE THREAD

**Issue by xzhangeb** [PaddlePaddle/PaddleOCR#2099]
Hi, I notice that you have released the training dataset for SRN, which consists of three files: aug_rotate_vgg_synth90k, aug_vgg_synth90k_300w, aug_vgg_synthtext240w. To reproduce the paper results, should we train only with these datasets, or should we combine them with original synth90k and synthtext? Here I see only 240w images for synthtext, is it all you use for training SRN?

**Comment by tink2123** [comment]
Yes, you need to combine the raw data（synth90k and synthtext） with the disturbed data（aug_rotate_vgg_synth90k, aug_vgg_synth90k_300w, aug_vgg_synthtext240w） to train the SNR model.

**Comment by vincentzn** [comment]
Thanks for proving the dataset! One question is it seems there are no label files included in the dataset? Only the lmdb files with keys like "image-000000000000" and values (bytes of image)
